tests/phpunit/tests/general-template/get-calendar-test.php

```diff
@@ -0,0 +1,522 @@
+<?php
+
+/**
+ * Class responsible for testing the functionality of the get_calendar() template function
+ * in various scenarios, including when a cached calendar is available, when no posts exist,
+ * current month rendering, and more.
+ *
+ * @covers get_calendar()
+ */
+
+class Tests_General_Template_GetCalendar extends WP_UnitTestCase {
+	/**
+	 * Tests if get_calendar() returns the cached calendar when available.
+	 *
+	 * This test function verifies that the get_calendar() function correctly
+	 * retrieves and returns a cached calendar when one is available in the cache.
+	 * It also tests the non-display mode of get_calendar().
+	 *
+	 * @ticket 41011
+	 *
+	 * @global string $m        The month in YYYYMM format.
+	 * @global string $monthnum The month number (01-12).
+	 * @global string $year     The year in YYYY format.
+	 *
+	 * @return void
```

Comment:
No needs to add doc for `@global` and `@return void` so remove it from all unit tests